From 7ec42740a45b21bca859cde5b7cbe2f09ef3d586 Mon Sep 17 00:00:00 2001
From: Dave Stevenson <dave.stevenson@raspberrypi.com>
Date: Thu, 28 Sep 2023 18:40:36 +0100
Subject: [PATCH] dt: Add DSI1 and DSI2 aliases to 2712

In order to keep the DRM names consistent as DSI-1 and DSI-2, add
aliases to the Pi5 DT.

Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.com>
---
 arch/arm/boot/dts/bcm2712-rpi-5-b.dts | 2 ++
 1 file changed, 2 insertions(+)

--- a/arch/arm/boot/dts/bcm2712-rpi-5-b.dts
+++ b/arch/arm/boot/dts/bcm2712-rpi-5-b.dts
@@ -785,6 +785,8 @@ spi10_cs_pins: &spi10_cs_gpio1 {};
 		gpio4 = &pinctrl_aon;
 		usb0 = &rp1_usb0;
 		usb1 = &rp1_usb1;
+		drm_dsi1 = &dsi0;
+		drm_dsi2 = &dsi1;
 	};
 
 	__overrides__ {
